src/compiler/build_tables/build_parse_table.cc

```diff
@@ -177,7 +177,7 @@ class ParseTableBuilder {
             parse_table.add_terminal_action(state_id, lookahead, action);
           } else {
             ParseAction &existing_action = entry.actions[0];
-            if (allow_any_conflict) {
+            if (existing_action.type == ParseActionTypeAccept || allow_any_conflict) {
```

Comment:
Pretty much nothing; it's fixing an unrelated bug. But the integration test that I wrote for epsilon rules crashed because it triggered this edge case, so I'm just including this as a drive-by fix.